anthos-bm-apigee/README.md

```diff
@@ -0,0 +1,81 @@
+
+## Apigee Hybrid on Anthos Baremetal on Google Compute Engine VMs with Terraform
+
+This repository is a sample Apigee Hybrid installation on Anthos Cluster on bare metal running on GCE Virtual Machine. This also assumes that Apigee organization name is same as the GCP Project Id. The installation will also create a trial Apigee Edge organization with same name as project id if  Apigee Organization is not available.
```

Comment:
- `This repository is a sample` -> `This is a sample`
- `Virtual Machine` -> `Virtual Machines`

Reply:
Making Changes